pom.xml

```diff
@@ -1424,6 +1425,7 @@
       <properties>
         <scala.version>${scala12.version}</scala.version>
         <scala.binary.version>2.12</scala.binary.version>
+        <skip.hudi-spark3.unit.tests>true</skip.hudi-spark3.unit.tests>
```

Comment:
whats the expected behavior if we run maven with both spark3 and scala12 ?

Reply:
I think we don't need to add `-Dscala2.12` for the spark3 build, since the scala2.12 was already hardcoded in spark3 profile.